tests/functional/BatchQueriesDisabledCept.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+$I = new FunctionalTester( $scenario );
+$I->wantTo( 'Test batch queries return errors when batching is disabled' );
+
+$options = [
+	'batch_queries_enabled' => 'off'
+];
+
+$I->haveOptionInDatabase( 'graphql_general_settings', $options );
+
+$settings = $I->grabOptionFromDatabase( 'graphql_general_settings' );
+
+$I->haveHttpHeader( 'Content-Type', 'application/json' );
+
+$I->sendPost( 'http://localhost/graphql', json_encode([
+	[
+		'query' => '{posts{nodes{id,title}}}',
+	],
+	[
+		'query' => '{posts{nodes{id,uri}}}'
+	]
+]));
+
+$I->seeResponseCodeIs( 200 );
```

Comment:
200 return code on query error?

Reply:
Yes. Since GraphQL as a spec isn't tied to any specific transport, HTTP error codes aren't part of the spec and errors are determined _within_ the response, not as part of the HTTP status.

This allows GraphQL to be used beyond HTTP (websockets, for example) because it can behave the same regardless of transport. 

This is a good explanation: https://stackoverflow.com/questions/59729656/http-status-code-handling-in-graphql-apis#59729656

I think there's room for adjustment in some cases, but I think we'd need to really think through the why's of each case and come up with some sort of documented standard if we did want to change this for WPGraphQL specifically.